CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ project(HTTPFsExtension)
 
 add_extension_definitions()
 
-include_directories(extension/httpfs/include ${DUCKDB_MODULE_BASE_DIR}/third_party/httplib)
+include_directories(extension/httpfs/include
```

Comment:
The CMakeList changes are made by linter, config change introduced at https://github.com/duckdb/duckdb-httpfs/pull/25